Add descriptions (#279)
stackabletech/commons-operator: docs/modules/commons-operator/pages/pod-enrichment.adoc

= Pod enricher

> :description: Automatically enrich Pods with Node address info using Stackable Commons Operator by enabling the enrichment label.

Reviewer: ```suggestion
:description: Use Labels to instruct the Stackable Commons Operator to enrich Pods with Kubernetes node addresses
```
